SegmentEditorFloodFilling/SegmentEditorFloodFillingLib/SegmentEditorEffect.py

```diff
@@ -75,28 +75,28 @@ def createCursor(self, widget):
     #return slicer.util.mainWindow().cursor
     return qt.QCursor(qt.Qt.PointingHandCursor)
 
-  def masterVolumeNodeChanged(self):
-    # Force recomputation of clipped master image data
-    self.clippedMasterImageData = None
+  def sourceVolumeNodeChanged(self):
```

Comment:
Keep backward compatibility with Slicer-5.0 to make it easier to keep the extension up-to-date for both stable and preview releases.